articles/quickstart/native/ios-swift/interactive.md

```diff
@@ -136,14 +136,14 @@ You can also configure the SDK programmatically. Check the [README](https://gith
 
 ::::checkpoint
 :::checkpoint-default
-The Auth0.swift SDK should now be properly configured. Run your app to verify that it is not producing any errors related to the SDK.
+You configured the Auth0.swift SDK. Run your app to verify that it is not producing any errors related to the SDK.
 :::
 
 :::checkpoint-failure
-Sorry about that. Here's a couple things to double check:
-- Make sure the correct Auth0 application is selected.
-- Did you press the **Save** button after entering your URLs?
-- Make sure the Auth0 Domain and Client ID are set correctly.
+If your application did not launch successfully:
```

Comment:
Same: the app launch is not what could fail here. The app will launch successfully.